bionic/code_hasher.py

```diff
@@ -219,6 +219,19 @@ def _update_hash(self, hash_accumulator, obj, code_context):
                 obj_bytes=self._check_and_hash(obj.value, code_context),
             )
 
+        elif isinstance(obj, (property, types.DynamicClassAttribute)):
+            if isinstance(obj, list):
```

Comment:
What's the relationship between being a property and being a list?

Reply:
This should have been a property 🤦🏽 . I'll see if I can add a test where property is replaced with dynamic class attr.

Reply:
I added a test for it and it fails without fixing this bug. Good catch!